posix.mak

```diff
@@ -152,21 +152,21 @@ DPL_DOCS_PATH=dpl-docs
 DPL_DOCS=$(DPL_DOCS_PATH)/dpl-docs
 REMOTE_DIR=d-programming@digitalmars.com:data
 TMP?=/tmp
+GENERATED=.generated
+G=$(GENERATED)
```

Comment:
Shall we just use G and delete GENERATED?

Reply:
I don't have a strong opinion about this, having it like this allows people to overwrite the more expressive `GENERATED` variable and doesn't cost us too much.
Note that I just moved these two lines up, s.t. the variables are in order (i.e. the are defined before used).

If you or someone else has a stronger opinion about this, let me know and then I will make a PR ;-)